platform/android/file_access_filesystem_jandroid.cpp

```diff
@@ -198,7 +200,7 @@ String FileAccessFilesystemJAndroid::get_line() const {
 			if (elem == '\n' || elem == '\0') {
 				// Found the end of the line
 				const_cast<FileAccessFilesystemJAndroid *>(this)->seek(start_position + line_buffer_position + 1);
-				if (result.parse_utf8((const char *)line_buffer.ptr(), line_buffer_position)) {
+				if (result.parse_utf8((const char *)line_buffer.ptr(), line_buffer_position, true)) {
```

Comment:
Line 211 may also need to be updated.

When following the logic it shouldn't need to be since any newline and carriage return would be detected / handled in this block, but I'd rather play it safe in case I'm overlooking an edge case.

Reply:
Fixed :+1: